adapters/platform_linux.c

```diff
@@ -7,6 +7,8 @@
 #include "azure_c_shared_utility/httpapiex.h"
 #ifdef USE_OPENSSL
 #include "azure_c_shared_utility/tlsio_openssl.h"
+#else
+const IO_INTERFACE_DESCRIPTION* tlsio_openssl_get_interface_description();
```

Comment:
wouldn't this need the definition which is in the header?

Reply:
only if USE_OPENSSL is used, which will include it in tlsio_openssl.h"

Reply:
If this is used for fuzzing, should it be called something other than openssl? Or is it actually somehow using openssl?

Reply:
note this is needed because of line 61